hlx_statics/styles/styles.css

```diff
@@ -950,4 +951,8 @@ main .action-buttons div{
 main .default-content-wrapper img{
   width: 100%;
   height: 100%;
-}
\ No newline at end of file
+}
+
+main .code-wrapper{
```

Comment:
I think we can move this to code.css (https://github.com/AdobeDocs/adp-devsite/blob/main/hlx_statics/blocks/code/code.css#L1), since `code-wrapper` will only exist if a `code` block exists